### DISCUSSION TITLE
Is full SD1.5 support coming to Fooocus?

### DISCUSSION THREAD

**Discussion by kubilaykilinc** [lllyasviel/Fooocus#788] [Ideas]
In the latest version of Fooocus MRE, full SD1.5 support was available. In the latest version of Fooocus, SD1.5 is only available if it is used as a refiner. There are still many quality models coming out of SD 1.5, and they will continue to come out. Fooocus is very easy to use. While I can get good results with the same model in Automatic 1111 and ComfyUI, I can generally get the most artistic results in Fooocus. It is also very fast when doing prototype work.

Is it possible for Fooocus to also have full SD1.5 support?

**Comment by lllyasviel** [comment]
Hi, we enourage users to use SD1.5 [using this tool](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
